fastify.js

```diff
@@ -420,6 +420,11 @@ function build (options) {
     if (!handler && typeof options === 'function') {
       handler = options
       options = {}
+    } else if (!handler && typeof options !== 'object') {
```

Comment:
`!handler` is repeated 3 times. Could you have a single `if (!handler) {` and then nest everything inside of that `if` statement?